docs/alerts.md

```diff
@@ -20,10 +20,82 @@
 ## Elasticsearch Cluster Health is Red
 
 At least one primary shard and its replicas are not allocated to a node.
+The most common causes can be failed cluster nodes and the elasticsearch process crashing due to a continuous heavy processing load.
 
 ### Troubleshooting
 
-Cluster nodes could be failed or the Elasticsearch process crashes due to heavy load.
+1. Confirm if the elasticsearch cluster’s health is RED:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_cluster_health?pretty
```

Comment:
drop the `?pretty` since that will cause this command to fail

Reply:
Done